public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -14,19 +13,20 @@ import { DashboardScene } from './DashboardScene';
 import { NavToolbarActions } from './NavToolbarActions';
 
 export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardScene>) {
-  const { controls, viewPanelKey: viewPanelId, overlay } = model.useState();
+  const { controls, viewPanelKey, overlay, editview } = model.useState();
   const styles = useStyles2(getStyles);
   const location = useLocation();
   const navIndex = useSelector((state) => state.navIndex);
   const pageNav = model.getPageNav(location, navIndex);
-  const bodyToRender = model.getBodyToRender(viewPanelId);
+  const bodyToRender = model.getBodyToRender(viewPanelKey);
+  const navModel = getNavModel(navIndex, 'dashboards/browse');
```

Comment:
This nav model is wrong. It shows Dashboards section nav instead of dashboard settings section nav.

Reply:
Uhm, but given this note :

```
This is assuming the future layout / design of dashboard settings that you see when you enable the dockedMegaMenu feature toggle. (IE dashboard settings with horizontal tabs).
```

this may actually be fine. So with docked mega menu there will be no section nav for dashboard settings? 🤔

Reply:
@dprokop no, just horizontal tabs